app/assets/stylesheets/scaffolds.scss

```diff
@@ -0,0 +1,84 @@
+body {
+  background-color: #fff;
+  color: #333;
+  margin: 33px;
+  font-family: verdana, arial, helvetica, sans-serif;
+  font-size: 13px;
+  line-height: 18px;
+}
```

Comment:
These extra scss files screw things up. Please delete the new ones. We will be only using application.scss